File: C++/Vetores/Abaixo_da_media.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int N, i;
    double media;

    cout << "Quantos elementos vai ter o vetor ? ";
    cin >> N;

    double vet[N];

    for (i = 0; i < N; i++)
    {
        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    double soma = 0;
    for (i = 0; i < N; i++)
    {
        soma = soma + vet[i];
    }

    media = soma / N;

    cout << endl;
    cout << "MEDIA DO VETOR: " << fixed << setprecision(3) << media << endl;

    cout << "ELEMENTOS ABAIXO DA MEDIA: " << endl;

    for (i = 0; i < N; i++)
    {
        if (vet[i] < media)
        {
            cout << fixed << setprecision(1);
            cout << vet[i] << endl;
        }
    }

    return 0;
}

File: Python/Vetores/Abaixo_da_media.py
N = int(input("Quantos elementos vai ter o vetor? "))

vet = [0 for x in range(N)]

for i in range(0, N):
    vet[i] = float(input("Digite um numero: "))

soma = 0

for i in range(0, N):
    soma = soma + vet[i]

media = soma / N

print()
print(f"MEDIA DO VETOR = {media:.3f}")

print ("ELEMENTOS ABAIXO DA MEDIA: ")

for i in range(0, N):
    if vet[i] < media:
        print(vet[i])
